### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by FireController1847**
Just a couple of changes here and there that I thought would make it easier to understand / read.

* Fix a bunch of small grammatical errors here and there, adding `or`'s and `and`'s appropriately.
* Add brackets around `options` on the main ytdl function as they are, of course, optional.
* Link to node's documentation page for readable stream.
* In the options, moved the description before any extra information. Obviously seen with the `filter` option, although the diff's for a couple other options were odd.
* Instead of saying `Note: xxxx`, use a sub-point.
* Remove quotes around events. I thought about using the ticks, but that looked odd, and seeing as we already have a colon indicating the name of it the quotes seem unnecessary.
* Link to the type for responses on events, verses just saying "Object".
* Change from `Number` to `number` as that's the correct "type" (`Number` is a constructor, not a type).
* Remove Stream#destroy as this is a ReadableStream feature and doesn't necessarily need to be explicitly stated (unless you manually add it in the code, in which tell me and I'll re-add it).
* For downloadFromInfo, fix `getInfo` to `ytdl.getInfo` (type specification) and generalize to "this function" verses saying the name of the function again.
* For chooseFormat and filterFormats, add a missing semicolon and put comments saying they're examples.
* For filterFormats, move the description above the example.
* Rename `install` to `installation`.
* Make the installation block a bash script, even though not completely necessary for consistency with the `npm test` script right below.

I'd love to hear your opinion on some of these things and if they're necessary or not. I'm ready to make changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,97 +20,96 @@ ytdl('http://www.youtube.com/watch?v=A02s8omM_hI')
...
-Emitted when the a video's `info` hash is fetched. Along with the chosen format metadata to download. `format.url` might be different if `start` was given.
+Emitted when the a video's `info` hash is fetched, along with the chosen format metadata to download. `format.url` might be different if `start` was given.
 
-An example of what Info and format may look like is in the [example folder](example/info.json).
+#### Event: response
+* [`http.ServerResponse`](https://nodejs.org/api/http.html#http_class_http_serverresponse) - Response.
 
-#### Event: 'response'
-* `http.ServerResponse` - Response.
+Emitted when the video response has been found and has started downloading. Can be used to get the size of download. This is also emitted if there is an error with the download or it needs to reconnect to YouTube.
```
Comment (fent) [inline README.md]: I know you didn't change this sentence but, while we're here,

"The size of **the** download."

It's not emitted if there's an error. Would clarify that it's emitted after reconnecting

File: README.md [hunk 2]
```diff
@@ -146,10 +145,11 @@ These tests are not mocked, and they actually try to start downloading a few vid
 For getting started with that, you can look at the `extractActions()` function in [`/lib/sig.js`](https://github.com/fent/node-ytdl-core/blob/master/lib/sig.js).
 
 
-# Install
-
-    npm install ytdl-core
+# Installation
```
Comment (fent) [inline README.md]: But I like the short noun (almost a noun) version :(

File: README.md [hunk 3]
```diff
@@ -20,97 +20,96 @@ ytdl('http://www.youtube.com/watch?v=A02s8omM_hI')
...
-Emitted when the a video's `info` hash is fetched. Along with the chosen format metadata to download. `format.url` might be different if `start` was given.
+Emitted when the a video's `info` hash is fetched, along with the chosen format metadata to download. `format.url` might be different if `start` was given.
 
-An example of what Info and format may look like is in the [example folder](example/info.json).
+#### Event: response
+* [`http.ServerResponse`](https://nodejs.org/api/http.html#http_class_http_serverresponse) - Response.
 
-#### Event: 'response'
-* `http.ServerResponse` - Response.
+Emitted when the video response has been found and has started downloading. Can be used to get the size of the download. This is not emitted if there is an error with the download or it needs to reconnect to YouTube, although it is emitted after a successful reconnect.
```
Comment (fent) [inline README.md]: Just the part of the last sentence is fine and can be moved to the first sentence, no need to say when it's not emitted.

    Emitted when the video response has been found and has started downloading, and after any successful reconnect.

Comment (FireController1847) [reply README.md]: Ok I am about to sleep but I'll make the change tomorrow :)

**Comment by fent** [comment]
Thanks again!

**Comment by FireController1847** [comment]
No problem! And don't forget to push these commits to NPM when you're ready 😝

**Pull request merged**